docs/en/samples/dataset_scaling.md

```diff
@@ -0,0 +1,184 @@
+# Demo - Cache Runtime Manual Scaling 
+
+## Prerequisites
+
+- [Fluid](https://github.com/fluid-cloudnative/fluid)(version >= 0.5.0)
+
+Please refer to the [Fluid installation](../userguide/install.md) to complete the installation
+
+## Set Up Workspace
+```shell
+$ mkdir <any-path>/dataset_scale
+$ cd <any-path>/dataset_scale
+```
+
+## Running
+
+**Create Dataset and AlluxioRuntime resource objects**
+```yaml
+$ cat << EOF > dataset.yaml
+apiVersion: data.fluid.io/v1alpha1
+kind: Dataset
+metadata:
+  name: hbase
+spec:
+  mounts:
+    - mountPoint: https://mirrors.tuna.tsinghua.edu.cn/apache/hbase/stable/
+      name: hbase
+---
+apiVersion: data.fluid.io/v1alpha1
+kind: AlluxioRuntime
+metadata:
+  name: hbase
+spec:
+  replicas: 1
+  tieredstore:
+    levels:
+      - mediumtype: MEM
+        path: /dev/shm
+        quota: 2Gi
+        high: "0.95"
+        low: "0.7"
+EOF
+```
+
+In the above example, we set `AlluxioRuntime.spec.replicas` to 1, which means we will start an Alluxio cluster with one worker to cache the data in the dataset.
+
+```
+$ kubectl create -f dataset.yaml
+dataset.data.fluid.io/hbase created
+alluxioruntime.data.fluid.io/hbase created
+```
+After the Alluxio cluster has started normally, we can see that the Dataset and AlluxioRuntime created before are in the following state:
+
+Status of the Alluxio components:
+```
+$ kubectl get pod
+NAME                 READY   STATUS    RESTARTS   AGE
+hbase-master-0       2/2     Running   0          3m50s
+hbase-worker-0       2/2     Running   0          3m15s
+```
+
+Status of the Dataset：
+```
+$ kubectl get dataset hbase
+NAME    UFS TOTAL SIZE   CACHED   CACHE CAPACITY   CACHED PERCENTAGE   PHASE   AGE
+hbase   544.77MiB        0.00B    2.00GiB          0.0%                Bound   3m28s
+```
+
+Status of the AlluxioRuntime:
+```
+$ kubectl get alluxioruntime hbase -o wide
+NAME    READY MASTERS   DESIRED MASTERS   MASTER PHASE   READY WORKERS   DESIRED WORKERS   WORKER PHASE   READY FUSES   DESIRED FUSES   FUSE PHASE   AGE
+hbase   1               1                 Ready          1               1                 Ready          0             0               Ready        4m55s
+```
+
+**Scale-up Dataset**
```

Comment:
Moreover, since we have the title `Cache Runtime Manual Scaling`, I think better to avoid using "Dataset" and "Scale" together

Reply:
> I think `Scale out` is better here? Scaling up normally indicates upgrading the requested resources of one Pod, but we are attempting to involve more pods now.

Actually I use 'Scale-out' at the first time, but I find that [the configuration of HPA](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/#configurable-scaling-behavior) uses 'ScaleUp' and 'ScaleDown', so I use them too. But I also think it is more accuracy to use 'Scale-out' and 'Scale-in'.


> Moreover, since we have the title `Cache Runtime Manual Scaling`, I think better to avoid using "Dataset" and "Scale" together

About rename the title to `Cache Runtime Manually Scaling`, I find that in our other document (e.g. dataset auto scaling) also use 'Scaling Dataset'. I rethink this title and I have a question: if an user is not quite familiar with Fluid, what will come into his brain when he wants to 'scale something'? Dataset or Cache Runtime?